plasmapy/atomic/classes.py

```diff
@@ -90,20 +96,21 @@ def __init__(self,
             self._lepton_number = _Particles[particle]['lepton number']
             self._baryon_number = _Particles[particle]['baryon number']
             self._integer_charge = _Particles[particle]['charge']
-            self._mass = _Particles[particle]['mass']
-
-            self._is_stable = _Particles[particle]['is_stable']
-
+            self._mass = _Particles[particle].get('mass', None)
```

Comment:
I'm not sure, but if the only thing you expect to get in these cases is an exact numerical value or `None` if that's not in there... make sure you're aware of `collections.defaultdict`!

Reply:
I'll look into that!